includes/User.php

```diff
@@ -228,6 +235,9 @@ class User {
 	 */
 	private $mBlockedFromCreateAccount = false;
 
+	/** @var UserPreferences */
+	private $preferences;
```

Comment:
Minor: I would prefer to call this something like `$preferenceService`. Something with "service" in the name so it's clear what we are working with.